src/spdx/parser/tagvalue/lexer.py

```diff
@@ -107,7 +107,7 @@ class SPDXLexer(object):
                  "UNKNOWN_TAG",
                  "ORGANIZATION_VALUE",
                  "PERSON_VALUE",
-                 "DATE",
+                 "ISO8601_DATE",
```

Comment:
As there was recently a [discussion](https://github.com/spdx/ntia-conformance-checker/issues/95) about the date not being exatly ISO8601 I am wondering if we should really name it like that.

Reply:
Let's be clear: it **is** an ISO-8601 formatted date.

It doesn't allow _all_ variants of ISO-8601, which include `20230328T122341.123` or `20230328T12XXXX` or `2023-33`. You would find very few implementations handling all these variants.